app/server.go

```diff
@@ -53,7 +53,8 @@ func main() {
 	pflag.StringP("confpath", "c", "config/", "Configuration file path")
 	pflag.BoolP("debug", "d", false, "Enable debug mode")
 	pflag.Bool("version", false, "Print version")
-
+        pflag.StringP("var-dir", "var-dir","/var/lib/glutton", "Set var-dir")
```

Comment:
You are using spaces instead of tab for indentation, this tells me that you are still not using gofmt correctly...
Also if you don't use the short flag, just use pflag.String